wolfhsm/wh_server.h

```diff
@@ -12,6 +12,7 @@
 #include "wolfhsm/wh_common.h"
 #include "wolfhsm/wh_comm.h"
 #include "wolfhsm/wh_nvm.h"
+#include "wolfhsm/wh_message_custom.h"
```

Comment:
Minor nit.  Recommend move the WH_CUSTOM_RQST_NUM_CALLBACKS enum/#define into here to avoid any message includes here, unless there was another reason that message_custom.h was included here.  Fix at your convenience.